### DISCUSSION TITLE
Empty Layout

### DISCUSSION THREAD

**Discussion by mabdalla123** [abpframework/abp#18930] [Q&A] [answered]
iam trying to use Empty layout but i cant access  jquery or localization 
my end goal is there are pages i want to create  a diffrent layout and use it 
how can do it , recommendation for this  process

**Comment by maliming** [comment, marked as answer]
hi

Please add your js/css bundles.

https://docs.abp.io/en/abp/latest/UI/AspNetCore/Bundling-Minification

https://github.com/abpframework/abp/blob/dev/modules/basic-theme/src/Volo.Abp.AspNetCore.Mvc.UI.Theme.Basic/Themes/Basic/Layouts/Application.cshtml#L46

https://github.com/abpframework/abp/blob/dev/modules/basic-theme/src/Volo.Abp.AspNetCore.Mvc.UI.Theme.Basic/Themes/Basic/Layouts/Application.cshtml#L76

**Reply by mabdalla123** [reply]
thank you
